scala/bigtable-shc/README.md

```diff
@@ -71,45 +89,47 @@ Create environment variables for the following commands:
     BIGTABLE_TABLE=my-table
     SPARK_CLUSTER=spark-cluster
 
-## Build the jar
+## Building (Assembling) Example
 
-The Spark job is assembled into a uber/fat jar with all of its dependencies and hbase configuration. To build, run:
+The Spark application is assembled into an uber/fat jar with all of its dependencies and hbase configuration. To build, run:
 
-    mvn assembly:assembly
+    mvn clean assembly:assembly
 
-Note that since the Bigtable configuration is included in the fat jar, any changes
+The above command should build `target/cloud-bigtable-dataproc-spark-shc-0.1-jar-with-dependencies.jar` file.
+
+**NOTE**: Since the Bigtable configuration is included in the fat jar, any changes
  will require a repackaging of the uberjar.
 
-## Test your job locally (optional but recommended)
+## Testing Locally (Optional but Recommended)
```

Comment:
I think the old one was better?
or maybe "Test locally"

@sduskis should we remove the optional but recommended parenthetical? Might as well just have people do it by default and they can decide on their own to